editor/editor_file_system.cpp

```diff
@@ -532,7 +532,12 @@ bool EditorFileSystem::_update_scan_actions() {
 				String full_path = ia.dir->get_file_path(idx);
 				if (_test_for_reimport(full_path, false)) {
 					//must reimport
-					reimports.push_back(full_path);
+					List<String> deps;
+					ResourceLoader::get_dependencies(full_path, &deps);
+
+					for (List<String>::Element *E = deps.front(); E; E = E->next()) {
+						reimports.push_back(E->get());
+					}
```

Comment:
I think you could simplify with this:
```
Vector<String> deps = _get_dependencies();
reimports.append_array(deps);
```

Reply:
just  `_get_dependencies` and not `_get_dependencies(full_path)` ?

Reply:
Er yes, with `full_path`.